ci-operator/step-registry/optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-ref.yaml

```diff
@@ -0,0 +1,35 @@
+ref:
+  as: optional-operators-ci-run-bundle
+  from_image:
+    namespace: ci
+    name: cvp-operator-scorecard
+    tag: v1
+  commands: optional-operators-ci-run-bundle-commands.sh
+  resources:
+    requests:
+      cpu: 300m
+      memory: 300Mi
+  dependencies:
+  - name: bundle-image
+    env: OO_BUNDLE_IMG
+  env:
+  - name: OO_INSTALL_NAMESPACE
+    default: "create"
```

Comment:
Let's be consistent with the other step and prefix the special value with an exclamation mark. For consistency's sake, and also to allow actually calling a namespace `create` ;)